feat(timezone): implement timezone picker on profile settings

genie-magic opened the pull request:
**What this PR does** 📖
implement timezone picker on profile settings

**Which issue(s) this PR fixes** 🔨

AP-518

**Special notes for reviewers** 🗒️
- @WanderingHogan , @stavares843  for getting utcoffset you can use getUtfOffsetInMins() in Utility/Timezone.ts
-  We should approve PR [on locales repo](https://github.com/Satellite-im/locales/pull/26) first because i18n text is added

stavares843 commented:
@genie-magic 

thanks for adding

i know you added the locales on the other repo but we removed the submodule so you need to add the locale on the main repo on this file
[https://github.com/Satellite-im/Core-PWA/blob/dev/locales/en-US.js](https://github.com/Satellite-im/Core-PWA/blob/dev/locales/en-US.js?rgh-link-date=2022-02-04T19%3A14%3A26Z)

stavares843 commented:
/rebase

genie-magic commented:
> @genie-magic
> 
> thanks for adding
> 
> i know you added the locales on the other repo but we removed the submodule so you need to add the locale on the main repo on this file [https://github.com/Satellite-im/Core-PWA/blob/dev/locales/en-US.js](https://github.com/Satellite-im/Core-PWA/blob/dev/locales/en-US.js?rgh-link-date=2022-02-04T19%3A14%3A26Z)

@stavares843  I added.
